js/whitebit.js

```diff
@@ -1252,7 +1262,7 @@ module.exports = class whitebit extends Exchange {
             // api error to be solved
             price = undefined;
         }
-        if (side === 'buy' && type.indexOf ('market') >= 0) {
+        if (side === 'buy' && type === 'market') {
```

Comment:
@DoctorSlimm, what if the type is 'stock market' or 'stop_market'?

Reply:
@carlosmiei it does not apply to `stock_market` actually, the api is strange

Reply:
@DoctorSlimm yeah that's what I've noted on that printscreen, we only need to add `stop market` to the if condition

Reply:
@carlosmiei I dont understand what you mean, it currently works as expected

Reply:
@DoctorSlimm If you look at the amount in the last screenshot (9.99), that is wrong, it should be the cost because when the raw type is `stop market` this condition `if (side === 'buy' && type === 'market') {` will be false

is it clear now?